### PR TITLE
Docs: Buffer: code style update

### PR DESCRIPTION
**Pull request by mtmccrea**
## Purpose and Motivation

This is a trivial documentation style change to illustrate the PR process for a tutorial. So I might as well submit it :)

## Types of changes

- Documentation

## To-do list

- [x] Code is tested
- [x] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

File: HelpSource/Classes/Buffer.schelp [hunk 1]
```diff
@@ -687,8 +687,8 @@ s.boot;
 a = Array.fill(2000000,{ rrand(0.0,1.0) });
```
Comment (JordanHendersonMusic) [inline HelpSource/Classes/Buffer.schelp:687]: missing spaces after comma

File: HelpSource/Classes/Buffer.schelp [hunk 2]
```diff
@@ -329,10 +329,10 @@ This allows for larger collections than setn, below. This is not as safe as link
 code::
 s.boot;
 (
-a = Array.fill(2000000,{ rrand(0.0,1.0) }); // a LARGE collection
-b = Buffer.sendCollection(s, a, 1, 0, {arg buf; "finished".postln;});
+a = Array.fill(2000000, { rrand(0.0,1.0) }); // a LARGE collection
```
Comment (JordanHendersonMusic) [inline HelpSource/Classes/Buffer.schelp:332]: and here inside the `rrand`

File: HelpSource/Classes/Buffer.schelp [hunk 3]
```diff
@@ -256,7 +256,7 @@ discussion::
 code::
 s.boot;
 (
-SynthDef(\help_Buffer_cue,{ arg out=0,bufnum;
+SynthDef(\help_Buffer_cue,{ |out = 0, bufnum|
 	Out.ar(out,
 		DiskIn.ar( 1, bufnum )
```
Comment (JordanHendersonMusic) [inline HelpSource/Classes/Buffer.schelp:261]: This line (261) has extra spaces in.

File: HelpSource/Classes/Buffer.schelp [hunk 4]
```diff
@@ -215,18 +215,18 @@ code::
 // with a completion message
 s.boot;
 (
-SynthDef(\help_Buffer,{ arg out=0, bufnum;
+SynthDef(\help_Buffer,{ |out = 0, bufnum|
```
Comment (JordanHendersonMusic) [inline HelpSource/Classes/Buffer.schelp:218]: A missing space after comma

File: HelpSource/Classes/Buffer.schelp [hunk 5]
```diff
@@ -215,18 +215,18 @@ code::
...
 	Out.ar( out,
```
Comment (JordanHendersonMusic) [inline HelpSource/Classes/Buffer.schelp:219]: here is and extra space after the parentheses.

**Review by JordanHendersonMusic — CHANGES_REQUESTED** [review]
Hi, thanks for this, and the upcoming documentation! 

This looks good, but I think you have missed a couple of spaces.

**Review by JordanHendersonMusic — APPROVED** [review]

**Comment by JordanHendersonMusic** [comment]
> all the code style issues

Across all the docs? ...that sounds like a herculean task! Might be worth asking on the forum if other people might be interested in take a file or two - good chance to get some new contributors?

Do you think I should merge this? The wiki says a third person isn't needed in the case of doc changes like this, and that new reviewers shouldn't merge (which I am).

**Comment by mtmccrea** [comment]
> Across all the docs? 

Haha no... _but_ funny you should mention it, I have slipped down a bit of a rabbit hole with search and replace throughout the docs and it seems somewhat feasible. I've already [swept a few rules through](https://github.com/mtmccrea/supercollider/tree/topic/docs-code-style-update) a few hundred files—it's not foolproof, and not fully automatic, but doable I think. If it's looking ok after a bit more hacking, I may post it as a PR just for feedback to gauge interest.

> Do you think I should merge this?

Let's hold off for now, I think "Trusted Reviewers" are the ones that merge (I don't do this myself). I've been meaning to post a question on the forum Development Topic to get some clarity on these details (and update the wiki with the info...)

**Review by adcxyz — APPROVED** [review]
looks good!
in lines 259 and 296, you are still missing a space between comma and curly brace:
SynthDef(\help_Buffer_cue,{ |out = 0, bufnum|

**Comment by mtmccrea** [comment]
Thanks for catching those! Updated.

**Comment by mtmccrea** [comment]
Since there are two approvals on this, I'll go rogue and merge my own PR 😈

**Pull request merged**